apps/system/js/app_chrome.js

```diff
@@ -7,12 +7,16 @@
 /* global SettingsListener */
 /* global Service */
 /* global GaiaPinCard */
+/* global Icon */
+/* global devicePixelRatio */
 
 'use strict';
 
 (function(exports) {
   const DEFAULT_ICON_URL = '/style/chrome/images/default_icon.png';
   const PINNING_PREF = 'dev.gaia.pinning_the_web';
+  // 32px + 5px padding added by the Icon renderer
+  const ICON_SIZE = (32 + 4) / devicePixelRatio;
```

Comment:
IconsHelper.getIcon also multiplies sizes by the devicePixelRatio, at https://github.com/mozilla-b2g/gaia/blob/master/shared/js/icons_helper.js#L49 so I'm not sure you need this here